### ISSUE TITLE
do client need to replicate whole gbs of db ?

### ISSUE THREAD

**Issue by yashwanth2804** [orbitdb/orbit-db#402]
does replicating bulk dbs resources and time consuming, anyway can we limit upto few records  
thanks

**Comment by RichardLitt** [comment]
Good question, @yashwanth2804. 

Let's make sure to add an answer to the docs when someone provides one.

**Comment by mistakia** [comment]
I don't believe there is currently a way to limit replication. You may want to explore modeling your data differently (on `orbit-db` or even `ipfs`).

Generally speaking, when modeling data for OrbitDB you should strive to "partition" it where it makes sense - i.e. along access rights. As an example, in a twitter-like application, each user should have their own database instead of a global "tweets" database for everyone.

**Comment by mistakia** [comment]
If you have the tweets locally - you can limit how many entries are loaded by sending in an `amount` to `db.load(amount)`:
```javascript
const db = await orbitdb.feed("<justin's tweet OrbitDB address>")
await db.load(10)
// db now contains the 10 latest entries/tweets
```

However, if you don't have the data locally, I'm not sure there's a way to limit syncing to only the 10 latest entries.

**Comment by cristiano-belloni** [comment]
If you can't limit how many data you can replicate, it means that malicious peers could trick you to "follow" them, then just flood you with data. Which might be a huge problem in 99% of the use cases.

**Comment by aphelionz** [comment]
Moving to Field Manual to discuss further
